contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_list_to_be_unique.py

```diff
@@ -121,9 +120,15 @@ def _validate(
                         }
                     },
                     "out": {"success": False},
-                    "only_for": ["sqlite"],
                 },
             ],
+            "test_backends": [
```

Comment:
Can you delete the whole `test_backends` section, then add `"suppress_test_for": ["postgresql", "bigquery"],` after line 100

```
--- a/contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_list_to_be_unique.py
+++ b/contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_list_to_be_unique.py
@@ -98,6 +98,7 @@ class ExpectQueriedColumnListToBeUnique(QueryExpectation):
                     },
                 },
             ],
+            "suppress_test_for": ["postgresql", "bigquery"],
             "tests": [
                 {
                     "title": "basic_positive_test",
@@ -122,13 +123,6 @@ class ExpectQueriedColumnListToBeUnique(QueryExpectation):
                     "out": {"success": False},
                 },
             ],
-            "test_backends": [
-                {
-                    "backend": "sqlalchemy",
-                    "dialects": ["sqlite"],
-                },
-                {"backend": "spark", "dialects": None},
-            ],
         }
     ]
```